draft-ietf-quic-transport.md

```diff
@@ -310,6 +306,13 @@ The two type bits from a Stream ID therefore identify streams as summarized in
 
 The first bidirectional stream opened by the client is stream 0.
 
+If an endpoint receives a frame for a stream that it expects to initiate (i.e.,
+an even-numbered for the client or an odd-numbered for the server), but which it
+has not yet opened, it MUST close the connection with error code
+STREAM_STATE_ERROR.  Similarly, receiving a STREAM or RST_STREAM frame for a
```

Comment:
This also applies to STREAM_DATA_BLOCKED.

Reply:
Yeah, I thought of that.  Now I'm regretting having this paragraph.  Maybe it's best to just remove it and rely on other text.

Reply:
Just add a note: Other misguided stream frames such as MAX... also closes the cinnection.